spec/defines/foreman_proxy_plugin_spec.rb

```diff
@@ -12,7 +12,13 @@
       end
 
       context 'no parameters' do
-        package = facts[:osfamily] == 'Debian' ? 'ruby-smart-proxy-myplugin' : 'rubygem-smart_proxy_myplugin'
+        package = if facts[:osfamily] == 'Debian'
+                    'ruby-smart-proxy-myplugin'
+                  elsif facts[:osfamily] == 'RedHat'
```

Comment:
Can/want we also test RHEL8 w/o SCL here?

Reply:
If you limit this to EL7, EL8 should hit the else branch:
```suggestion
                  elsif facts[:osfamily] == 'RedHat' && facts[:operatingsystemmajrelease] == '7'
```